Xcode/BZFlag.xcodeproj/project.pbxproj

```diff
@@ -7309,9 +7309,13 @@
 					../include,
 					../Xcode,
 					/usr/local/include,
+					/opt/homebrew/include,
```

Comment:
Is this a change in Homebrew behavior? They used to install everything in /usr/local.

Reply:
Yea, see my review. For M1 devices, they've moved installation to `/opt/homebrew/`

Reply:
well that's unfortunately inconsistent. And it appears that Homebrew abandoned IRC (there used to be a freenode channel that was relatively active).

https://docs.brew.sh/Installation shows:
* `/usr/local` for macOS Intel
* `/opt/homebrew` for Apple Silicon
* `/home/linuxbrew/.linuxbrew` for Linux

Oh well. Looks good to me too.

Reply:
I wonder if this should (could) be contextual? That is, use /opt/homebrew for M1 and /usr/local for Intel?
https://docs.brew.sh/FAQ#why-is-the-default-installation-prefix-opthomebrew-on-apple-silicon

Reply:
Alternatively, `/opt/homebrew` should be before `/usr/local`, so you find native libraries first. Although I'm not sure that's an alternative. As this PR stands, things could be found in either directory, leading to confusing mixed architecture build errors. Especially with the preference for `/usr/local`.

Reply:
Thank you both for your helpful comments. The header and library paths should now depend on the architecture. My knowledge of Xcode is very (very) limited though. At least on my machine the current solution seems to work.